dds_web/__init__.py

```diff
@@ -194,6 +194,12 @@ def prepare():
             # Verify cli version compatible
             if "api/v1" in flask.request.path:
                 verify_cli_version(version_cli=flask.request.headers.get("X-Cli-Version"))
+            elif "api/v3" in flask.request.path:  # experimental v3 version
+                # If version api is not provided, it gets the data from the __version__ file
+                # When v3 is finallized, this should be removed and the __version__ file should be updated
+                verify_cli_version(
+                    version_cli=flask.request.headers.get("X-Cli-Version"), version_api="3.0.0"
+                )
```

Comment:
The first time I was using the v3 the requests failed if this was not specified because the versions mismatch. Now that I rethink about it, maybe I could make it work with just a pass instruction?